src/lib-voyager.tsx

```diff
@@ -36,7 +45,8 @@ export class Voyager {
   private data: Data;
   private filename: string;
 
-  constructor(container: Container, config: VoyagerConfig, data: Data) {
+  constructor(container: Container, params: VoyagerParams = DEFAULT_VOYAGER_PARAMS) {
```

Comment:
Of the top of my head, I think we don't really need the default `DEFAULT_VOYAGER_PARAMS` as the store already has default `data` and `config`?   (Let me know if I'm wrong as my memory is not perfect.)

Note that if we really need it, we probably better use
```
const {config=DEFAULT_VOYAGER_CONFIG, data=DEFAULT_DATA} = params;
```
so the default is always used if only one parameter is specified.